executor/join_test.go

```diff
@@ -2456,6 +2457,87 @@ func (s *testSuiteJoinSerial) TestIssue20270(c *C) {
 	failpoint.Disable("github.com/pingcap/tidb/executor/killedInJoin2ChunkForOuterHashJoin")
 }
 
+func (s *testSuiteJoinSerial) TestIssue20710(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	tk.MustExec("drop table if exists t;")
+	tk.MustExec("drop table if exists s;")
+	tk.MustExec("create table t(a int, b int)")
+	tk.MustExec("create table s(a int, b int, index(a))")
+	tk.MustExec("insert into t values(1,1),(1,2),(2,2)")
+	tk.MustExec("insert into s values(1,1),(2,2),(2,1)")
+	tk.MustQuery("explain select /*+ inl_join(s) */ * from t join s on t.a=s.a and t.b = s.b").Check(testkit.Rows(
```

Comment:
Could we move the plan tests to test data?

Reply:
+1, that way make it eaiser to change the test content later.